src/CardList.tsx

```diff
@@ -14,28 +22,102 @@ interface CardListProps {
   removeBlock: (id: number) => void;
 }
 
+const theme = createTheme({
+  palette: {
+    primary: {
+      main: "#fff",
+    },
+    secondary: {
+      main: "#000",
+      light: "#333",
+    },
+  },
+  spacing: 5,
+});
+
 const CardList: React.FC<CardListProps> = ({ blocks, removeBlock }) => {
   return (
-    <div className="cardList">
-      {blocks.map((block) => (
-        <div key={block.id} className="card">
-          <h2 className="cardTitle">Match Info:</h2>
-          <label className="cardLabel">
-            {block.firstTeam} - {block.secondTeam}
-          </label>
-          <label className="cardLabel">
-            Number of tickets - "{block.tickets}"
-          </label>
-          <label className="cardLabel">Field - "{block.stadium}"</label>
-          <button
-            onClick={() => removeBlock(block.id)}
-            className="deleteCardButton"
+    <ThemeProvider theme={theme}>
+      <Box
+        sx={{
+          margin: 0,
+        }}
+      >
+        {blocks.map((block) => (
+          <Grid
+            key={block.id}
+            sx={{
+              display: "flex",
+              flexDirection: "column",
+              borderRadius: theme.spacing(6),
+              width: theme.spacing(70),
+              backgroundColor: "primary.main",
+              boxShadow: 5,
+              marginTop: theme.spacing(2),
+            }}
           >
-            <span className="button-content">Delete</span>
-          </button>
-        </div>
-      ))}
-    </div>
+            <Typography
+              sx={{
+                marginTop: theme.spacing(1),
+                fontSize: theme.spacing(4),
+                fontFamily: "Platypi",
+                fontWeight: 600,
+              }}
+            >
+              Match Info:
+            </Typography>
+            <Typography
+              sx={{
+                fontFamily: "Platypi",
+                fontSize: theme.spacing(3),
+                marginTop: theme.spacing(2),
+              }}
+            >
+              {block.firstTeam} - {block.secondTeam}
+            </Typography>
+            <Typography
+              sx={{
+                fontFamily: "Platypi",
+                fontSize: theme.spacing(3),
+                marginTop: theme.spacing(2),
+              }}
+            >
+              Number of tickets - "{block.tickets}"
+            </Typography>
+            <Typography
+              sx={{
+                fontFamily: "Platypi",
+                fontSize: theme.spacing(3),
+                marginTop: theme.spacing(2),
+              }}
+            >
+              Field - "{block.stadium}"
+            </Typography>
+            <Button
+              onClick={() => removeBlock(block.id)}
```

Comment:
Can you override styles for buttons in the theme? You can have a look at Ivan`s theme if there is any problems.
You can add the styles that are used for all buttons and not duplicate them